test/solo/notifications.test.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+// oboe's notifier will send all log messages
+process.env.APPOPTICS_DEBUG_LEVEL = 6;
+
+const ao = require('../..');
```

Comment:
Is there some magic here? You can just require the root directory?

Reply:
maybe i've been missing something all along but how do i know what the root directory of the package is?

Reply:
I mean '../..' wouldn't that be the root/top directory of the repo?

Reply:
it is, but how else can i refer to the root/top of the repo?